docs/src/examples.md

```diff
@@ -2,543 +2,180 @@
-- ``g_1(\rho, q) = wD_h(G_h(q))``
-- ``DSS(\rho, q) = DSS(g_1(\rho q))``
-- ``g_2(\rho, q) = -\nu_4 wD_h(\rho G_h(\rho q))``
-    - with ``\nu_4`` the hyperviscosity coefficient.
+The Ekman depth parameter is calculated as `d = sqrt(2 * ν / f)`, which determines the characteristic depth of the boundary layer.
 
-#### Application of Flux Limiters
+The initial condition is set to a uniform wind profile equal to the geostrophic wind throughout the vertical column. The simulation is run for 50 hours to allow the boundary layer to develop fully.
 
-!!! note
+Boundary conditions are applied as follows:
+- At the top boundary: wind velocity equals the geostrophic wind (`u_g`, `v_g`)
+- At the bottom boundary: drag condition proportional to the wind speed, where the surface stress is `Cd * |U| * u` and `Cd * |U| * v`
 
-    Since we use flux limiters that limit only operators defined in the spectral space (i.e., they are applied level-wise in the horizontal direction), the application of limiters has to follow a precise order in the sequence of operations that specifies the total tendency.
+#### Application of boundary conditions
 
-The order of operations should be the following:
+The application of boundary conditions is implemented through the operators:
 
-1. Horizontal transport (with strong divergence ``D_h``)
-2. Horizontal Flux Limiters
-3. Horizontal hyperdiffusion (with weak divergence ``wD_h``)
-4. Vertical transport
-5. DSS
-#### Problem flow and set-up
+1. For the u-momentum equation:
+   - Top boundary: `Operators.SetValue(FT(ug))` sets u to the geostrophic value
+   - Bottom boundary: `Operators.SetValue(Geometry.WVector(Cd * u_wind * u_1))` applies the drag condition
 
-This test case is set up in a Cartesian (box) domain `` [-2 \pi, 2 \pi]^2 \times [0, 4 \pi] ~\textrm{m}^3``, doubly periodic in the horizontal direction, but not in the vertical direction.
+2. For the v-momentum equation:
+   - Top boundary: `Operators.SetValue(FT(vg))` sets v to the geostrophic value
+   - Bottom boundary: `Operators.SetValue(Geometry.WVector(Cd * u_wind * v_1))` applies the drag condition
 
-The flow was chosen to be a spiral, i.e., so to have a horizontal uniform rotation, and a vertical velocity ``\boldsymbol{u}_v \equiv w = 0`` at the top and bottom boundaries, and ``\boldsymbol{u}_v \equiv w = 1`` in the center of the domain. Moreover, the flow is reversed in all directions halfway through the time period so that the tracer blobs go back to its initial configuration (using the same speed scaling constant which was derived to account for the distance travelled in all directions in a half period).
+3. DSS (Direct Stiffness Summation) is applied implicitly through the operators to ensure numerical stability at element boundaries
```

Comment:
I don't think we should mention the DSS operator here since it does not appear explicitly in the example. But just for your own reference sake, I would have described it this way:

```suggestion
3. DSS (Direct Stiffness Summation) is applied implicitly through the operators to ensure continuity of the solution at element boundaries and reduction of unnecessary multiplicity
```